Sources/Sentry/include/SentryFileManager.h

```diff
@@ -15,6 +15,12 @@ NS_ASSUME_NONNULL_BEGIN
 @class SentryOptions;
 @class SentrySession;
 
+#if SENTRY_TEST || SENTRY_TEST_CI
+BOOL isErrorPathTooLong(NSError *error);
+
+BOOL createDirectoryIfNotExists(NSString *path, NSError **error);
+#endif // SENTRY_TEST || SENTRY_TEST_CI
+
```

Comment:
`l`: You could also move this to `SentryFileManager+Test.h` to keep making things visible for testing approach consistent for this class.